### DISCUSSION TITLE
Can I import from arc browser?

### DISCUSSION THREAD

**Discussion by sarim2000** [zen-browser/desktop#947] [Q&A]
I don't see it as an option, is there a way to do this?

**Comment by ahgraber** [comment]
AFAIK Arc itself does not support exporting so I imagine this would be a not insignificant feature

**Reply by jackTabsCode** [reply]
You'd have to parse their settings file, which is in a proprietary format. Though, I would find this feature super useful. I imagine that there's lots of Arc users that love the browser, but are dissuaded by it being closed source, Chromium, and barely cross-platform.

**Reply by ghost** [reply]
> You'd have to parse their settings file, which is in a proprietary format. Though, I would find this feature super useful. I imagine that there's lots of Arc users that love the browser, but are dissuaded by it being closed source, Chromium, and barely cross-platform.

Totally agree. Looking forward to a complete solution.
